frontend/assets/style/_layout.scss

```diff
@@ -0,0 +1,41 @@
+.l-no-navigation{
```

Comment:
Missing space before brace (just trying to keep things consistent and pretty 😄 )